service/history/replicatorQueueProcessor.go

```diff
@@ -105,11 +111,17 @@ func newReplicatorQueueProcessor(shard ShardContext, historyCache *historyCache,
 	return processor
 }
 
-func (p *replicatorQueueProcessorImpl) process(qTask queueTaskInfo) (int, error) {
+func (p *replicatorQueueProcessorImpl) getTaskFilter() queueTaskFilter {
+	return p.replicationTaskFilter
+}
+
+func (p *replicatorQueueProcessorImpl) process(qTask queueTaskInfo, shouldProcessTask bool) (int, error) {
```

Comment:
To make it clear that shouldProcessTask is not used can we write:

`func (p *replicatorQueueProcessorImpl) process(qTask queueTaskInfo, _ bool) (int, error) {`